routes/chat.js

```diff
@@ -107,20 +68,56 @@ router.post('/chat', async function (req, res) {
       is_group: false,
       messages: [],
     });
-    const chatID = chat._id;
+    const chatId = chat._id;
 
     await UserChat.create({
       user: user._id,
-      chat: chatID,
+      chat: chatId,
     });
     await UserChat.create({
       user: otherUser._id,
-      chat: chatID,
+      chat: chatId,
     });
 
-    res.json(chat);
+    res.status(201).json(chat);
   } else {
     // join
-    res.status(204).json(commonChats);
+    res.status(200).json(commonChats);
   }
 });
+
+router.post('/chat/:chatId/messages', async function (req, res) {
+  if (!ObjectId.isValid(req.params.chatId)) {
+    return res.status(406).end();
+  }
+
+  const message = req.body.message;
+  if (
+    message?.type === undefined ||
+    message?.content === undefined ||
+    message.content.length === 0 ||
+    !Object.values(MessageType).includes(message?.type)
+  ) {
+    return res.status(400).end();
+  }
+
+  const result = await UserChat.findOne({
+    chat: new ObjectId(req.params.chatId),
+    user: new ObjectId(req.userId),
+  });
+  console.log(result);
```

Comment:
provide better log:
string consisting of `route function paramsused - short message`